peer/brontide.go

```diff
@@ -2111,52 +2111,94 @@ func (p *Brontide) queueHandler() {
 	lazyMsgs := list.New()
 
 	for {
-		// Examine the front of the priority queue, if it is empty check
-		// the low priority queue.
+		// Process the priority queue first. We will not accept any new
+		// messages until this queue is empty.
 		elem := priorityMsgs.Front()
-		if elem == nil {
-			elem = lazyMsgs.Front()
-		}
 
+		// Found an item, we will write it to lnwire.
 		if elem != nil {
-			front := elem.Value.(outgoingMsg)
+			front, ok := elem.Value.(outgoingMsg)
+			if !ok {
+				p.log.Errorf("Unknown message type found in "+
```

Comment:
return err?

Reply:
+1, should we proceed with sending if the msg type is unknown?